R/tags.R

```diff
@@ -1757,7 +1757,7 @@ includeText <- function(path) {
 #' @rdname include
 #' @export
 includeMarkdown <- function(path) {
-  html <- markdown::markdownToHTML(path, fragment.only=TRUE)
+  html <- markdown::markdownToHTML(path, options = 'fragment_only')
```

Comment:
I think we'll probably want to do something like  

```r
html <- if (is_available("markdown", "TBD") {
  markdown::markdownToHTML(path, options = 'fragment_only')
} else {
  markdown::markdownToHTML(path, fragment.only=TRUE)
}
```

Since `{markdown}` is only a Suggests?

Reply:
Good idea.